### PR TITLE
Wrap pre element so it doesn't overflow

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Fixes https://issues.redhat.com/browse/AAH-599

Collection detail documentation page of: `ibm` -> `power_ibm` -> `module > ibmi_install_product_from_savf`

Before:
Screenshot 2021-06-11 at 10 30 49

After:
Screenshot 2021-06-11 at 10 42 45

### CODE REVIEW

**Review by ShaiahWren — COMMENTED** [review]
LGTM 
white-space selectors are new for me! It looks like it works well for the task at hand.

**Review by ShaiahWren — APPROVED** [review]
Nicely done.

File: src/containers/collection-detail/collection-detail.scss [hunk 1]
```diff
@@ -47,3 +47,6 @@
     }
   }
 }
+pre {
```
Comment (himdel) [inline src/containers/collection-detail/collection-detail.scss:50]: global style for every single `pre`? That doesn't sound safe, use a class?

Comment (ZitaNemeckova) [reply src/containers/collection-detail/collection-detail.scss:50]: Cannot use the class. It's generated in `@ansible/galaxy-doc-builder`

Comment (himdel) [reply src/containers/collection-detail/collection-detail.scss:50]: You can still use the known parent class, 
`.docs-container pre` is still better than global `pre` :)

**Comment by ZitaNemeckova** [comment]
@himdel Updated. Please have a look. Thanks :)

**Review by himdel — APPROVED** [review]
Works for me :)

**Pull request merged**